src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -564,14 +565,36 @@ public static void assertFileIsReadable(final File file) {
     public static void assertFileIsReadable(final Path path) {
         Utils.nonNull(path);
 
-        if ( ! Files.exists(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It doesn't exist.");
-        }
-        if ( ! Files.isRegularFile(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It isn't a regular file");
-        }
-        if ( ! Files.isReadable(path) ) {
-            throw new UserException.CouldNotReadInputFile(path, "It is not readable, check the file permissions");
+        try {
```

Comment:
I don't think we need anything special at the top beyond what's already there. Those exceptions do not includes the path, no, except possibly in the message itself (and we already print that).